cli/stream_command.go

```diff
@@ -192,7 +192,7 @@ func configureStreamCommand(app commandHost) {
 		if !edit {
 			f.Flag("storage", "Storage backend to use (file, memory)").EnumVar(&c.storage, "file", "f", "memory", "m")
 		}
-		f.Flag("compression", "Compression algorithm to use (file storage only)").Default("none").EnumVar(&c.compression, "none", "s2")
+		f.Flag("compression", "Compression algorithm to use (file storage only). Defaults to 'none'").EnumVar(&c.compression, "none", "s2")
```

Comment:
you could instead use `IsSetByUser()` to determine if the value was given on the cli or should be left alone, thats the pattern used here in other places so prefer we do that for consistency.

Reply:
👍